components/Modal/CreateRecipe.js

```diff
@@ -107,17 +132,27 @@ class CreateRecipe extends React.Component {
               <h4 className="modal-title" id="myModalLabel">Create Recipe</h4>
             </div>
             <div className="modal-body">
-              {this.state.inlineError === true &&
+              {(this.state.inlineError && this.state.showErrorName) ?
                 <div className="alert alert-danger">
                   <span className="pficon pficon-error-circle-o"></span>
-                  <strong>Required information is missing.</strong>.
+                  <strong>Required information is missing.</strong>
                 </div>
+                :
+                <div></div>
+              }
+              {(this.state.inlineError && this.state.showErrorDuplicate) ?
+                <div className="alert alert-danger">
+                  <span className="pficon pficon-error-circle-o"></span>
+                  <strong>Specify a new recipe name.</strong>
+                </div>
+                :
+                <div></div>
```

Comment:
Should we be using the other syntax (i.e. `&&` and `||` instead of `:` and `?`) so that we don't have to include the empty `<div></div>`?